src/Photos/Enums.cs

```diff
@@ -256,7 +281,14 @@ public enum PHAssetResourceType : long
 		FullSizeVideo = 6,
 		AdjustmentData = 7,
 		AdjustmentBasePhoto = 8,
-		PairedVideo = 9
+		[iOS (9,1)]
+		PairedVideo = 9,
+		[Mac (10,15, onlyOn64 : true), iOS (10,0)]
+		FullSizePairedVideo = 10,
+		[Mac (10,15, onlyOn64 : true), iOS (10,0)]
+		AdjustmentBasePairedVideo = 11,
+		[Mac (10,15, onlyOn64 : true), iOS (13,0)]
```

Comment:
^ you need to have the `[TV]` too

Reply:
Isn't the TV attribute already included for that entire enum?

Reply:
yes, but the newer values where **not** available in `[TV (10,0)]` so they must be decorated again with the SDK version that added them

Reply:
Oh so are you saying that since the new ones say "API_AVAILABLE(macos(10.15), ios(10))" for example, this means that it is not available in TV?

Reply:
not quite, from macros Apple iOS if often equal to tvOS (if the framework is available to the later)
However we're stricter when using attributes, so we have to explicitly do a `[iOS (13,0)][TV (13,0)]` for the IDE to show the right information to customers

Reply:
Ah I was confused because I was thinking TV 10 was the newest one, sorry!

Reply:
still missing `[TV(13,0)]` here ?

Reply:
Okay I think I was still confused before because I was misunderstanding you, sorry. This one needs the TV 13 since it has the iOS 13 meaning this one is actually new where as the previous properties were actually already created in previous versions, but they just weren't in the file so those ones are Tv 10 and do not need a special attribute since the header includes it?